ssg/templates.py

```diff
@@ -30,38 +112,70 @@ def package_installed(data, lang):
     return data
 
 
+def package_removed(data, lang):
+    return data
+
+
+def service_disabled(data, lang):
```

Comment:
OVAL checks for some `services_enabled` rules became incomplete, this is because OVAL template for `service_disabled` has an `extend_definition` for package remove.

For these cases, these services/packages have an entry for the service and package in both `services_disabled` and `packages_removed.csv. But there is no `rule.yml` for `package_removed`.
Examples: `acpid`, `autofs`.

For most cases, `service_disabled` can mean `package_removed`, unless the package of the service brings in some dependencies. I don't know of any such case right now, but it could happen.

A workaround would be for `service_disabled` preprocessor to also "inject" `package_removed` template and also generate this definition. I don't think the current mechanism allows for that though, :(

Reply:
The OVAL template should be able to be handle to generate both, so we should merge them together. We can leverage the Jinja macros to not repeat ourselves.